flight/PiOS/Common/pios_mpu6000.c

```diff
@@ -51,6 +56,17 @@
 #endif
 #define MPU6000_SPI_LOW_SPEED               1000000
 
+// Private variables
```

Comment:
Can you share that information?  It looked like from our earlier analysis and the delay milliseconds in the table we'd be a little ahead on latency.